pkg/engine/engine.go

```diff
@@ -119,16 +118,25 @@ func validateDistro(cs *api.ContainerService) bool {
 	return true
 }
 
-func generateIPList(count int, firstAddr string) []string {
+// generateConsecutiveIPsList takes a starting IP address and returns a string slice of length "count" of subsequent, consecutive IP addresses
+func generateConsecutiveIPsList(count int, firstAddr string) ([]string, error) {
```

Comment:
I actually made the effort to refactor + test this function because it was appalling in its current state